bin/install.sh

```diff
@@ -50,7 +50,11 @@ pip install --no-cache-dir $pip_deps
 export PYTHONOPTIMIZE="$optimize"
 
 # Security upgrades
-pip install --no-cache-dir "psycopg2>=2.7" # ODOO-SA-2017-06-15-1
+if [ "$ODOO_VERSION" == "8.0" ] || [ "$ODOO_VERSION" == "9.0" ]; then
+    pip install --no-cache-dir "psycopg2==2.7.1"
+else
+    pip install --no-cache-dir "psycopg2==2.7.3.1"
+fi
```

Comment:
Instead of doing this, please just remove the separate psycopg2 installation stuff (see line 43 too).

This was a patch referring to https://github.com/odoo/odoo/issues/17914 because they didn't update upstream `requirements.txt` files.

After awhile, I'm now seeing that https://github.com/OCA/OCB/commit/6f8efea28cd29fd973ac6a50d7b93f0d73a725c6 is found in 8-11 branches, so we don't need to patch anything downstream now.

Reply:
You're right, it doesn't make sense to ignore the version from `requirements.txt` and manually install the exact same version. Changing it right now!